aiosparql/test_utils.py

```diff
@@ -78,7 +74,7 @@ def post(self, *args, **kwargs):
             self._closed = True
 
 
-class AioSPARQLTestCase(AioHTTPTestCase):
+class AioSPARQLTestCase(TestSPARQLClient):
```

Comment:
I don't think this was intended to be used that way.  TestSPARQLClient is a client class while AioSPARQLTestCase is a TestCase class for unittest. Plus I don't think this class is used anywhere, you can maybe remove it entirely.

Reply:
Yeah, I didn't do much in this area. Not sure what's best.